dev_guide/shared_memory.adoc

```diff
@@ -12,75 +12,55 @@ toc::[]
 == Overview
 
 There are two types of shared memory objects in Linux: System V and POSIX.
-The containers running in a pod share the IPC namespace of the pod infra container and so are able to share the System V shared memory objects. This document describes how they can also share POSIX shared memory objects.
+The containers in a pod share the IPC namespace of the pod infrastructure container and so are able to share the System V shared memory objects.
+This document describes how they can also share POSIX shared memory objects.
 
 
 == POSIX Shared Memory
-POSIX shared memory requires that a tmpfs be mounted at /dev/shm. The containers
- in a pod do not share their mount namespaces so we use volumes to mount in the same /dev/shm into each container in a pod.
+POSIX shared memory requires that a tmpfs be mounted at `/dev/shm`.
```

Comment:
@adellape  Thanks for the tip.  I was looking around for example file/dir markup, but obviously didn't look hard enough.